README.md

```diff
@@ -19,6 +11,15 @@ IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
 FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
 #############################################################################
 ```
+---
+Upcomming changes:
+- Crucible's roles have been migrated to the [Red Hat CI OCP Ansible Collection](https://github.com/redhatci/ansible-collection-redhatci-ocp) available for download at [Ansible Galaxy](https://galaxy.ansible.com/ui/repo/published/redhatci/ocp/).  
+- Crucible's playbooks will continue to work but as always, you are encouraged to fork crucible and maintain your own playbooks.
```

Comment:
Perhaps put out a call to anyone who wants to main them here?

Reply:
I kept adding and removing the call for stewardship. I'll put it back.